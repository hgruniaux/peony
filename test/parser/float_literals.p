# RUN: peony -fsyntax-only -verify %s

fn f() {
    let f0 = 0.0_;
    let f1 = 5_f32;
    let f2 = 254f64;
    let f3 = 8_2e+_986_;
    let f4 = 2.5e+3;
    let f5 = 32_.2_3e+23f64;
}
