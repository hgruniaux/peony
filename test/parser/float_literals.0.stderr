<source>:7:14: error: float literal too large
    7 |     let f3 = 8_2e+_986_;
      |              ~~~~~~~~~~
